content/en/docs/tasks/security/authentication/mtls-migration/index.md

```diff
@@ -106,6 +106,15 @@ command terminated with exit code 56
 sleep.legacy to httpbin.bar: 200
 {{< /text >}}
 
+Optionally, you can verify that traffic is encrypted by running `tcpdump` in a proxy container, but only in an Istio installation with `values.global.proxy.privileged=true`. And you then can determine whether traffic is in plaintext
```

Comment:
If you installed Istio with `values.global.proxy.privildeged=true`, you can use `tcpdump` to verify traffic is encrypted or not:

Reply:
done.